tests/test_transforms/test_encoders/test_mean_segment_encoder_transform.py

```diff
@@ -22,6 +22,36 @@ def test_mean_segment_encoder_transform(simple_df, transformed_simple_df):
     pd.testing.assert_frame_equal(transformed_df, transformed_simple_df)
 
 
+def test_subset_segments(simple_df):
+    train_df = simple_df
+    test_df = simple_df.loc[:, pd.IndexSlice["Omsk", :]]
+    transform = MeanSegmentEncoderTransform()
+
+    transform.fit(train_df)
+    transformed_test_df = transform.transform(test_df)
+
+    segments = sorted(transformed_test_df.columns.get_level_values("segment").unique())
+    features = sorted(transformed_test_df.columns.get_level_values("feature").unique())
+    assert segments == ["Omsk"]
+    assert features == ["exog", "segment_mean", "target"]
+
+
+def test_not_fitted_error(simple_df):
+    encoder = MeanSegmentEncoderTransform()
+    with pytest.raises(ValueError, match="The transform isn't fitted"):
+        encoder.transform(simple_df)
+
+
+def test_new_segments_error(simple_df):
+    train_df = simple_df.loc[:, pd.IndexSlice["Moscow", :]]
+    test_df = simple_df.loc[:, pd.IndexSlice["Omsk", :]]
+    transform = MeanSegmentEncoderTransform()
+
+    transform.fit(train_df)
+    with pytest.raises(ValueError, match="This transform can't process segments that weren't present on train data"):
+        _ = transform.transform(test_df)
```

Comment:
Isn't it enough to just call the method without assigning to the variable?

Reply:
May be we should try: https://docs.python.org/3/library/reprlib.html

Reply:
I've ment assigning to underscore at line 52.

Reply:
I think that in most tests we make assignment to underscore.